src/align/LinearAlignment.cpp

```diff
@@ -157,7 +157,7 @@ std::string OptimalTranscript(const std::string& target, int j1, int j2, const s
     //
     // Base case
     //
-    if ((j2 - j1 <= 1) || (i2 - i1 <= 1)) {
```

Comment:
I approve of this---though this didn't actually affect correctness, it just meant that the base case was kicking in earlier than it needed to.